web/source/kmwembedded.ts

```diff
@@ -56,10 +56,8 @@ namespace com.keyman.osk {
       if(key['subKeys'] && (typeof(window['oskCreatePopup']) == 'function')) {
         var xBase = dom.Utils.getAbsoluteX(key) - dom.Utils.getAbsoluteX(this.kbdDiv) + key.offsetWidth/2,
             yBase = dom.Utils.getAbsoluteY(key);
-        
-        if(util.device.formFactor == 'phone') {
-          this.prependBaseKey(key);
```

Comment:
Does this mean we can delete the `prependBaseKey` function?

Reply:
I thought about it, but opted to leave it to @jahorton to decide

Reply:
Per team discussion, we'll remove `prependBaseKey`